src/sqlfluff/dialects/dialect_spark3.py

```diff
@@ -1679,6 +1679,61 @@ class TransformClauseSegment(BaseSegment):
     )
 
 
+@spark3_dialect.segment(replace=True)
+class ExplainStatementSegment(BaseSegment):
+    """An `Explain` statement.
+
+    Enhanced from ANSI dialect to allow for additonal parameters.
+
+    EXPLAIN [ EXTENDED | CODEGEN | COST | FORMATTED ] explainable_stmt
+
+    https://spark.apache.org/docs/latest/sql-ref-syntax-qry-explain.html
+    """
+
+    type = "explain_statement"
+
+    explainable_stmt = ansi_dialect.get_segment(
+        "ExplainStatementSegment"
+    ).explainable_stmt.copy(
+        insert=[
+            Ref("AlterDatabaseStatementSegment"),
```

Comment:
Any reason this shouldn't just be all statements (i.e. `Ref("StatementSegment"),`)?

Reply:
Perfect! I had looked for something like that and missed it.

> Why are these not just Refs? like the rest?
From the comment above, Using `StatementSegment` would resolve that as well.